.circleci/config.yml

```diff
@@ -233,6 +233,18 @@ jobs:
       - store_artifacts: *artifacts_solc
       - persist_to_workspace: *artifacts_executables
 
+  b_ubu18: &build_ubuntu1804
+    docker:
+      - image: ethereum/solidity-buildpack-deps:ubuntu1804
+    environment:
```

Comment:
Why the extra flags? (compared to `b_ubu`)

Reply:
the compilation error in that I fixed in this PR as well is only triggered with those flags.

Reply:
Those are also the flags we use for the PPA release. At least the relevant ones.